packages/messaging/lib/index.d.ts

```diff
@@ -630,7 +706,7 @@ export namespace FirebaseMessagingTypes {
      *
      * @param options Options to override senderId (iOS) and projectId (Android).
      */
-    getToken(options?: TokenOptions): Promise<string>;
+    getToken(options?: GetTokenOptions): Promise<string>;
```

Comment:
I think a union type (assuming there was a new `NativeTokenOptions` type with senderId/appName members) could work like this ?

```suggestion
    getToken(options?: GetTokenOptions | NativeTokenOptions): Promise<string>;
```